README.md

```diff
@@ -92,10 +92,16 @@ If you enter a invalid Snyk token, a pop-up message alerts you that your Snyk to
 If you need a new Snyk token, you can generate a new token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&utm_source=code_ready&code_ready=FF1B53D9-57BE-4613-96D7-1D06066C38C9).
 
 **Red Hat Dependency Analytics Report File Path** :
-
-Specify the local path to create the Red Hat Dependency Analytics report file.
+<br >Specify the local path to create the Red Hat Dependency Analytics report file.
 The default path is `/tmp/redhatDependencyAnalyticsReport.html`.
 
+<br >**IMPORTANT:** 
+<br >The `redHatDependencyAnalyticsReportFilePath` setting name has changed to `reportFilePath`.
+If you had a custom file path set for `redHatDependencyAnalyticsReportFilePath`, then you need to add your custom file path to the `reportFilePath` setting.
+
+**Inline Vulnerability Severity Alerts** :
+<br >You can set the vulnerability severity alert level to receive inline notifications for just errors or warnings.
```

Comment:
This sentence might not be very clear, the notifications will remain the same no matter the setting, but the sentence should make it clear that you have the option to configure whether these notifications appear as errors or warnings through the settings.

Reply:
Thanks for the feedback!  I did a slight rewrite of the sentence, please see my latest commit.